src/common/snippets/include/snippets/utils/debug_caps_config.hpp

```diff
@@ -51,6 +51,7 @@ class DebugCapsConfig {
         std::string dir = "snippets_LIR_dump";
         LIRFormatFilter format = {1 << LIRFormatFilter::controlFlow};
         std::vector<std::string> passes;
+        std::string name_modifier;
```

Comment:
Do we really need to store this field as string? Maybe we can just have a boolean value `use_subgraph_name`?

Reply:
There are some reasons:
1. I assumed there could be some other customizations
2. We don't have `BoolPropertySetter`, we will need to introduce it.

Should we switch to bool?